.travis.yml

```diff
@@ -1,8 +1,3 @@
 language: java
-script: mvn test
-addons:
-  apt:
-    packages:
-      - openjdk-7-jdk
 jdk:
-  - openjdk7
+  - openjdk8
```

Comment:
I am not using JDK 7 in my apps since a long time but maybe someone still does.
Is there a reason to move to JDK 8 ?
If it builds fine with 7 maybe it could stay with it.
In pom.xml it says `1.6`: https://github.com/openjson/openjson/blob/29b2d19c7ed9295b8ed490d94436ac86bc395aa6/pom.xml#L58

Reply:
JDK8 supports source level 6/7 generation, so it looks like this is Travis file only change and no new release is needed?